basket/news/tasks.py

```diff
@@ -562,6 +563,16 @@ def upsert_contact(api_call_type, data, user_data):
         if settings.MAINTENANCE_MODE:
             sfdc_add_update.delay(update_data)
         else:
+            ctms_data = update_data.copy()
+            email_id = ctms_data["email_id"] = generate_token()
```

Comment:
Didn't realize basket would be generating CTMS email IDs.

Reply:
We can go either way, since it is a v4 UUID. This felt easier for the first code, but we should probably pick the general strategy going forward.

For this version, ``POST /ctms`` uses the ``email_id`` in the request body as the ID. The benefit is we don't have to parse the response, either the body or the Location header.

If we want CTMS to pick the ID, we can omit the ``email_id`` in the body, CTMS will generate one. We'll need to decide if ``CTMS.add()`` returns just the ``email_id``, the full JSON, or both.